build.gradle

```diff
@@ -39,9 +39,34 @@ String getConclaveRevision() {
     return conclave_revision
 }
 
+enum ArtifactType {
```

Comment:
I think VersionType is better name. ArtifactType implies we're dealing with different artifacts, whatever that might mean (jar, war, etc).

Reply:
Replaced